app/models/metric/ci_mixin/processing.rb

```diff
@@ -95,21 +96,26 @@ def write_metrics(parameters)
     ActiveMetrics::Base.connection.write_multiple(parameters)
   end
 
-  def update_and_publish_metrics(interval_orig, start_time, end_time, resources, rt_rows)
+  def update_last_perf_capture_on(resources, end_time)
```

Comment:
These extracted methods should probably be made private.

Reply:
thnx. made all of these methods private.